packages/@aws-cdk-testing/framework-integ/test/aws-codepipeline-actions/test/integ.pipeline-with-pipeline-triggers.ts

```diff
@@ -0,0 +1,61 @@
+import * as codebuild from 'aws-cdk-lib/aws-codebuild';
+import * as codepipeline from 'aws-cdk-lib/aws-codepipeline';
+import * as cdk from 'aws-cdk-lib';
+import { IntegTest } from '@aws-cdk/integ-tests-alpha';
+import * as cpactions from 'aws-cdk-lib/aws-codepipeline-actions';
+
+const app = new cdk.App();
+
+const stack = new cdk.Stack(app, 'aws-cdk-codepipeline-with-pipeline-triggers');
+
+const sourceOutput = new codepipeline.Artifact();
+// Make sure you specify a valid connection ARN.
+const connectionArn = process.env.CONNECTION_ARN || 'MOCK';
```

Comment:
I'm assuming you grabbed this from elsewhere in this module. Wonder if we can at least provide a synth-time warning here to alert people manually running this integ test. 

I sense that what usually happens is that I will run the integ test, it will fail at deploy time, and then I have to go searching for why. Having a warning show up on console would at least give me a fighting chance of identifying the issue pre-deploy. what do you think?